data/software.json

```diff
@@ -1994,6 +1953,17 @@
         ],
         "url": "https://www.xabber.com"
     },
+    {
+        "categories": [
+            "client"
+        ],
+        "doap": null,
+        "name": "xmpp",
```

Comment:
Does this make sense? shouldn't it be "xmpp for plan 9"

Reply:
Well, the program itself is literally just called “xmpp”.

Reply:
It's the project's name, so that's fine.